src/components/popup-menu/popup-menu.styles.js

```diff
@@ -0,0 +1,111 @@
+export default ({ palette, typography, mixins }) => ({
+  menuPopup: {
+    position: 'absolute',
+    display: 'inline-block',
+    top: 0,
+    right: 0,
+    background: palette.color.white,
+    boxShadow: '0 2px 4px 2px rgba(0, 0, 0, .1)',
+    listStyle: 'none',
+    padding: 15,
+    marginTop: 55,
+    zIndex: 10,
+
+    '&:before': {
+      position: 'absolute',
+      right: '4%',
+      transform: 'translateX(-50%) rotate(45deg)',
+      boxShadow: '-1px -1px 1px rgba(0, 0, 0, .08)',
+      display: 'block',
+      content: '""', // Need the extra inner "", otherwise stripped away
+      width: 0,
+      height: 0,
+      zIndex: 5,
+      top: -8,
+      borderLeft: `7px solid ${palette.color.white}`,
+      borderRight: `7px solid ${palette.color.white}`,
+      borderBottom: `7px solid ${palette.color.white}`,
+      borderTop: `7px solid ${palette.color.white}`,
+    },
+  },
+  menuContainer: {
+    position: 'relative',
+    display: 'flex',
+  },
+  menuItemContainer: {
+    display: 'block',
+    borderBottom: `1px solid ${palette.background.secondary}`,
+    fontSize: 12,
+    color: palette.text.secondary,
+
+    '&:last-child': {
+      borderBottom: 'none',
+    },
+  },
+  buttonMenuItem: {
+    color: palette.color.black,
+    fontFamily: typography.primary,
+    fontSize: 14,
+    background: 'none',
+    border: 'none',
+    cursor: 'pointer',
+    outline: 'none',
+    textAlign: 'left',
+  },
+  menuButton: {
+    color: palette.color.grayDarkest,
+    textDecoration: 'none',
+    background: 'none',
+    border: 'none',
+    cursor: 'pointer',
+    outline: 'none',
+  },
+  menuSlideDown: {
+    '&-enter': {
+      opacity: 0.01,
+      transform: 'translate(0, -5px)',
+
+      '&-active': {
+        opacity: 1,
+        transform: 'translate(0, 0)',
+        transitionProperty: 'transform, opacity',
+        transition: 'opacity 300ms ease-in',
```

Comment:
Create transitions using the built-in helper for consistency.
`transition: transitions.create(['opacity', 'transform']),`